tpkg/util.go

```diff
@@ -36,44 +41,70 @@ func RandByte() byte {
 func RandBytes(length int) []byte {
 	var b []byte
 	for i := 0; i < length; i++ {
-		b = append(b, byte(rand.Intn(127)))
+		b = append(b, RandByte())
 	}
 
 	return b
 }
 
 func RandString(length int) string {
-	return string(RandBytes(length))
+	var b []byte
+	for i := 0; i < length; i++ {
+		// 128 because we want to have printable ASCII characters
+		b = append(b, byte(RandInt(128)))
```

Comment:
If we want to have it printable we should limit it to `32` to `126`, because you don't want the control characters in a string. Or why not just generate a random alphanumeric string?